input/pagecontent/general-guidance.md

```diff
@@ -110,6 +110,295 @@ Example: Patient resource with interpreter required and language is known
+
+a) Body site with laterality is defined in a pre-coordinated primary finding/procedure code.
+
+b) Body site without laterality is defined in a primary finding/procedure code AND coded laterality qualifier is separate.
+
+c) Body site with laterality is defined separately from a primary finding/procedure code.
+
+d) Body site without laterality AND coded laterality qualifier are defined separately from a primary finding/procedure code.
+
+
+To support consistent representation the following is recommended for each of these cases, this approach can be applied to either Condition or Procedure profiles:
+
+a) Primary `code` only (pre-coordinated body site including laterality)
+* For systems that have pre-coordinated coding describing a concept fully.
+* Only the `code` element is used and contains information on body site with laterality.
+
+Example Condition - Cellulitis of right knee
+~~~
+{
+  "resourceType" : "Condition",
+  "id" : "cellulitis",
+  "text" : {
+    "status" : "generated",
+    "div" : "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><a href=\"Patient-bennelong-anne.html\">Patient/bennelong-anne: Anne Bennelong</a> " BENNELONG"</p></div>"
+  },
+  "clinicalStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-clinical",
+        "code" : "active"
+      }
+    ]
+  },
+  "verificationStatus" : {
+    "coding" : [
+      {
+        "system" : "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+        "code" : "confirmed"
+      }
+    ]
+  },
+  "category" : [
+    {
+      "coding" : [
+        {
+          "system" : "http://terminology.hl7.org/CodeSystem/condition-category",
+          "code" : "encounter-diagnosis",
+          "display" : "Encounter Diagnosis"
+        }
+      ]
+    }
+  ],
+  "code" : {
+    "coding" : [
+      {
+        "system" : "http://snomed.info/sct",
+        "code" : "10633311000119108",
+        "display" : "Cellulitis of right knee"
+      },
+      "text" : "Cellulitis of right knee"
+    ]
+  },
+  "subject" : {
+    "reference" : "Patient/bennelong-anne",
+    "display" : "Anne Bennelong"
+  }
+}
+~~~
+
+b) Primary `code` only (precoordinated body site without laterality and separate laterality qualifier)
+* For systems that have 
+  * Pre-coordinated coding describing a concept including body site without laterality
+  * Laterality qualifier recorded separately e.g. left, right
+* The `code` element is used
+  * Contains `coding` for primary concept (no body site information)
+  * Use `text` to describe concept fully, this can include information on recorded laterality e.g. ', Right'
+* NOTE: in this case laterality is NOT expressed in coded form 
+
+Example Condition - Cellulitis of knee, laterality as text only
```

Comment:
Replace "Example Condition - Cellulitis" with "Example Condition resource with cellulitis"

Apply this type change to the title for all example snippets

Reply:
@brettesler-ext  missing word "with" in the Example title

Reply:
done